docs/Installing-Clear-Containers-3.0-on-Fedora.md

```diff
@@ -0,0 +1,56 @@
+# Installing Clear Containers 3.0 Alpha on Fedora
+
+## Introduction
+
+Clear Containers **3.0-alpha** is available for Fedora\* versions **24** and **25**.
+
+## Install latest Docker
+
+This step is only required in case Docker is not installed on the system.
+
+```
+$ sudo dnf -y install dnf-plugins-core
+$ sudo dnf config-manager \
+    --add-repo \
+    https://download.docker.com/linux/fedora/docker-ce.repo
+$ sudo dnf makecache fast
+$ sudo dnf install docker-ce
+```
+
+For more information on installing Docker please refer to the
+[Docker Guide](https://docs.docker.com/engine/installation/linux/fedora)
+
+## Install Clear Containers 3.0 components
+
+```
+$ source /etc/os-release
+$ sudo -E dnf config-manager --add-repo \
```

Comment:
`-E` won't give `dnf` access to `$VERSION_ID` unless that variable is `export`ed, or you specify it on the `sudo` command-line:

```
$ sudo -E VERSION_ID=$VERSION_ID dnf ...
```